api/preprints/serializers.py

```diff
@@ -73,71 +73,73 @@ class PreprintSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
     id = IDField(source='_id', read_only=True)
     type = TypeField()
 
-    date_created = VersionedDateTimeField(source='created', read_only=True)
+    date_created = VersionedDateTimeField(source='created', allow_null=True, read_only=True)
```

Comment:
Why was this changed?

Reply:
probably confused`date_created`, `date_retracted` when adding `allow_null`. 😢